### ISSUE TITLE
ModuleNotFoundError:No module named 'llvmlite.llvmpy'

### ISSUE THREAD

**Issue by janzandr** [EnMAP-Box/enmap-box#791]
_Reported by @dpoursanidis via email._

Problem under macOS. Installation problems with numba package (required for EnSoMAP):

```
ModuleNotFoundError:No module named 'llvmlite.llvmpy'
Traceback:
File "/Users/manuellopezramirez/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/enmapboxplugin/enmapbox/gui/applications.py", line 365, in addApplicationFolder
apps = factory(self.mEnMAPBox)
File "/Users/manuellopezramirez/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/enmapboxplugin/enmapbox/apps/ensomap/__init__.py", line 35, in enmapboxApplicationFactory
from ensomap.enmapboxintegration import EnSoMAP
File "/Users/manuellopezramirez/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/ee_plugin/__init__.py", line 38, in __wrapping_ee_import__
_module_ = __builtin_import__(name, *args, **kwargs)
File "/Applications/QGIS.app/Contents/MacOS/../Resources/python/qgis/utils.py", line 888, in _import
mod = _builtin_import(name, globals, locals, fromlist, level)
File "/Users/manuellopezramirez/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/enmapboxplugin/enmapbox/apps/ensomap/enmapboxintegration.py", line 32, in
import hys
File "/Users/manuellopezramirez/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/ee_plugin/__init__.py", line 38, in __wrapping_ee_import__
_module_ = __builtin_import__(name, *args, **kwargs)
File "/Applications/QGIS.app/Contents/MacOS/../Resources/python/qgis/utils.py", line 888, in _import
mod = _builtin_import(name, globals, locals, fromlist, level)
File "/Users/manuellopezramirez/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/enmapboxplugin/enmapbox/apps/ensomap/hys/__init__.py", line 26, in
from .tools import *
File "/Users/manuellopezramirez/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/ee_plugin/__init__.py", line 38, in __wrapping_ee_import__
_module_ = __builtin_import__(name, *args, **kwargs)
File "/Applications/QGIS.app/Contents/MacOS/../Resources/python/qgis/utils.py", line 888, in _import
mod = _builtin_import(name, globals, locals, fromlist, level)
File "/Users/manuellopezramirez/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/enmapboxplugin/enmapbox/apps/ensomap/hys/tools.py", line 5, in
from numba import jit, float32, void, intc, int64, int32
File "/Users/manuellopezramirez/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/ee_plugin/__init__.py", line 38, in __wrapping_ee_import__
_module_ = __builtin_import__(name, *args, **kwargs)
File "/Applications/QGIS.app/Contents/MacOS/../Resources/python/qgis/utils.py", line 888, in _import
mod = _builtin_import(name, globals, locals, fromlist, level)
File "/Applications/QGIS.app/Contents/MacOS/lib/python3.9/site-packages/numba-0.50.1-py3.9-macosx-10.13.0-x86_64.egg/numba/__init__.py", line 34, in
from numba.core.decorators import (cfunc, generated_jit, jit, njit, stencil,
File "/Users/manuellopezramirez/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/ee_plugin/__init__.py", line 38, in __wrapping_ee_import__
_module_ = __builtin_import__(name, *args, **kwargs)
File "/Applications/QGIS.app/Contents/MacOS/../Resources/python/qgis/utils.py", line 888, in _import
mod = _builtin_import(name, globals, locals, fromlist, level)
File "/Applications/QGIS.app/Contents/MacOS/lib/python3.9/site-packages/numba-0.50.1-py3.9-macosx-10.13.0-x86_64.egg/numba/core/decorators.py", line 12, in
from numba.stencils.stencil import stencil
File "/Users/manuellopezramirez/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/ee_plugin/__init__.py", line 38, in __wrapping_ee_import__
_module_ = __builtin_import__(name, *args, **kwargs)
File "/Applications/QGIS.app/Contents/MacOS/../Resources/python/qgis/utils.py", line 888, in _import
mod = _builtin_import(name, globals, locals, fromlist, level)
File "/Applications/QGIS.app/Contents/MacOS/lib/python3.9/site-packages/numba-0.50.1-py3.9-macosx-10.13.0-x86_64.egg/numba/stencils/stencil.py", line 11, in
from numba.core import types, typing, utils, ir, config, ir_utils, registry
File "/Users/manuellopezramirez/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/ee_plugin/__init__.py", line 38, in __wrapping_ee_import__
_module_ = __builtin_import__(name, *args, **kwargs)
File "/Applications/QGIS.app/Contents/MacOS/../Resources/python/qgis/utils.py", line 888, in _import
mod = _builtin_import(name, globals, locals, fromlist, level)
File "/Applications/QGIS.app/Contents/MacOS/lib/python3.9/site-packages/numba-0.50.1-py3.9-macosx-10.13.0-x86_64.egg/numba/core/ir_utils.py", line 16, in
from numba.core.extending import _Intrinsic
File "/Users/manuellopezramirez/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/ee_plugin/__init__.py", line 38, in __wrapping_ee_import__
_module_ = __builtin_import__(name, *args, **kwargs)
File "/Applications/QGIS.app/Contents/MacOS/../Resources/python/qgis/utils.py", line 888, in _import
mod = _builtin_import(name, globals, locals, fromlist, level)
File "/Applications/QGIS.app/Contents/MacOS/lib/python3.9/site-packages/numba-0.50.1-py3.9-macosx-10.13.0-x86_64.egg/numba/core/extending.py", line 17, in
from numba.core.pythonapi import box, unbox, reflect, NativeValue # noqa: F401
File "/Users/manuellopezramirez/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/ee_plugin/__init__.py", line 38, in __wrapping_ee_import__
_module_ = __builtin_import__(name, *args, **kwargs)
File "/Applications/QGIS.app/Contents/MacOS/../Resources/python/qgis/utils.py", line 888, in _import
mod = _builtin_import(name, globals, locals, fromlist, level)
File "/Applications/QGIS.app/Contents/MacOS/lib/python3.9/site-packages/numba-0.50.1-py3.9-macosx-10.13.0-x86_64.egg/numba/core/pythonapi.py", line 6, in
from llvmlite.llvmpy.core import Type, Constant
File "/Users/manuellopezramirez/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/ee_plugin/__init__.py", line 38, in __wrapping_ee_import__
_module_ = __builtin_import__(name, *args, **kwargs)
File "/Applications/QGIS.app/Contents/MacOS/../Resources/python/qgis/utils.py", line 888, in _import
mod = _builtin_import(name, globals, locals, fromlist, level)
```

**Comment by jakimowb** [comment]
relates to https://trac.osgeo.org/osgeo4w/ticket/823

**Comment by jakimowb** [comment]
closed with #843
